setup.cfg

```diff
@@ -1,18 +1,116 @@
+[metadata]
+name = napari
+url = https://napari.org
+download_url = https://github.com/napari/napari
+license = BSD 3-Clause
+license_file = LICENSE
+description = n-dimensional array viewer in Python
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = napari team
+author_email = what_is_it@gmail.com  # FIXME
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: X11 Applications :: Qt
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: C
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Visualization
+    Topic :: Scientific/Engineering :: Information Analysis
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Topic :: Utilities
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: MacOS
+
+
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+include_package_data = True
+install_requires =
+    dask[array]>=2.1.0
+    cachey>=0.2.1
+    fsspec>=0.3.3
+    imageio>=2.5.0
+    Pillow!=7.1.0,!=7.1.1  # not a direct dependency, but 7.1.0 and 7.1.1 broke imageio
+    ipykernel>=5.1.1
+    numpy>=1.10.0
+    qtpy>=1.7.0
+    qtconsole>=4.5.1
+    scipy>=1.2.0
+    vispy>=0.6.4
+    IPython>=7.7.0
+    backcall>=0.1.0
+    PyOpenGL>=3.1.0
+    PyYAML>=5.1
+    wrapt>=1.11.1
+    psutil>=5.0
+    numpydoc>=0.9.2
+    napari-plugin-engine>=0.1.5
+    napari-svg>=0.1.3
+    tifffile>=2020.2.16
+    toolz>=0.10.0
+    importlib-metadata>=1.5.0 ; python_version < '3.8'
+
+
+[options.extras_require]
+pyside2 = PySide2>=5.12.3,<5.15.0
+pyqt5 = PyQt5>=5.12.3,<5.15.0
+tests = 
+    pandas
+    pytest
+    pytest-faulthandler
+    pytest-qt
+    pytest-ordering
+    pytest-timeout
+    xarray
+    zarr
+    scikit-image
+    pooch
+release = 
+    PyGithub>=1.44.1
+    twine>=3.1.1
+    pygithub
+    tqdm
+docs = 
+    Sphinx==3.0.1
+    sphinx-rtd-theme==0.4.3
+    recommonmark==0.6.0
+    scikit-image
+dev =
+    pre-commit>=1.16.1
+    black==19.10b0
+    flake8==3.7.9
+    check-manifest>=0.42
+    %(pyside2)s
+    %(tests)s
+    %(docs)s
+    
+
+
+[options.entry_points]
+console_scripts =
+    napari = napari.__main__:main
+
 
 [flake8]
 # Ignores - https://lintlyci.github.io/Flake8Rules
 # E203  Whitespace before ':'  (sometimes conflicts with black)
 # E501 line too long (84 > 79 characters)  (sometimes too annoying)
+# E741 ambiguous variable name
 # W503	Line break occurred before a binary operator
 # C901 McCabe complexity test. Would be nice to reenable, but takes work
-ignore = E203,W503,E501,C901
+ignore = E203,W503,E501,C901,E741
```

Comment:
Hmm, why?

Reply:
that was actually because flake8 started failing locally for me with that error.  I realize now I was using a newer version of flake8 (3.8.2).  If we keep it pinned to `3.7.9`, it runs fine without ignoring that error.  I'll remove it for now and we can delay that one for later (I think avoiding "ambiguous variable names" is a good rule... but it would require a lot of corrections in our repo)

Reply:
> it would require a lot of corrections

Wow, really! That'll be fun! =P

Reply:
if you're curious:

<details>

```
napari/conftest.py:193:47: E741 ambiguous variable name 'l'
napari/conftest.py:195:33: E741 ambiguous variable name 'l'
napari/layers/labels/labels.py:664:28: F541 f-string is missing placeholders
napari/_tests/test_mouse_bindings.py:90:23: E741 ambiguous variable name 'l'
napari/_tests/test_mouse_bindings.py:106:23: E741 ambiguous variable name 'l'
napari/_tests/test_mouse_bindings.py:162:23: E741 ambiguous variable name 'l'
napari/_tests/test_mouse_bindings.py:178:23: E741 ambiguous variable name 'l'
napari/_tests/test_key_bindings.py:89:22: E741 ambiguous variable name 'l'
napari/_tests/test_key_bindings.py:98:28: E741 ambiguous variable name 'l'
napari/plugins/exceptions.py:143:59: F541 f-string is missing placeholders
napari/plugins/_tests/test_pypi.py:29:5: F541 f-string is missing placeholders
napari/utils/_magicgui.py:108:28: E741 ambiguous variable name 'l'
napari/components/layerlist.py:60:13: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:127:24: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:130:24: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:133:24: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:147:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:151:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:153:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:157:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:173:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:175:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:178:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:199:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:203:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:214:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:226:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:237:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:240:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:264:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:270:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:274:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:280:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:284:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:287:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:297:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:302:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:307:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:312:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:317:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:322:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:327:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:330:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:335:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:341:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:347:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:353:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:359:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:376:9: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:389:28: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:423:27: E741 ambiguous variable name 'l'
napari/components/_tests/test_layers_list.py:427:27: E741 ambiguous variable name 'l'
napari/components/_tests/test_add_layers.py:22:42: E741 ambiguous variable name 'l'
napari/_qt/_tests/test_qt_layerlist.py:155:9: E741 ambiguous variable name 'l'
napari/_qt/_tests/test_qt_layerlist.py:205:9: E741 ambiguous variable name 'l'
```

</details>

Reply:
😱